assets/src/animation/constants.js

```diff
@@ -74,6 +74,28 @@ export const ANIMATION_EFFECTS = {
   },
 };
 
+/*
+ * @TODO See what Sam wants the user facing names of these animation
+ *       parts to be here & update `name` field.
+ */
```

Comment:
```suggestion
```

Per your note - think this comment is out of date and can be removed

Reply:
ahh. yeaup. thanks lol